src/DependencyInjection/DI.External.Tests/test/Autofac.cs

```diff
@@ -1,20 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
@maryamariyan I've removed all DI spec and external tests since they will be moved to runtime. If you need the source for reference you can find it in the commit history.